src/common/timing.h

```diff
@@ -41,8 +56,9 @@ int32_t ticks_diff(uint32_t ticks_a, uint32_t ticks_b);
 /// Given the datatype, overflows every ~584 years
 uint64_t timestamp_ns();
 
-/// To be called every 1 ms from TIM12 IRQ
-void tick_ms_irq();
+/// Sys timer's overflow interrupt callback
+///
+void TICK_TIMER_PeriodElapsedCallback();
```

Comment:
/question/ Why the uppercase letters? Why to change it to a name which does not fit the naming style of this module?

/suggestion/ I believe something like `tick_irq()` or `tick_timer_callback_irq()` would be more appropriate here.